activesupport/lib/active_support/core_ext/enumerable.rb

```diff
@@ -61,7 +62,8 @@ def index_by
     end
   end
 
-  # Convert an enumerable to a hash keying it with the enumerable items and with the values returned in the block.
+  # Convert an enumerable to a hash, using the element as the key and the block
+  # result as the value.
   #
   #   post = Post.new(title: "hey there", body: "what's up?")
   #
```

Comment:
Looks like we’re missing docs for the default constant param too. E.g. enum.index_with(0)

Reply:
Good spot, I've pushed an update which adds another example.